common/test/acceptance/tests/discussion/test_cohort_management.py

```diff
@@ -259,19 +259,29 @@ def test_add_new_cohort(self):
         Given I have a course with a user in the course
         When I add a new manual cohort to the course via the LMS instructor dashboard
         Then the new cohort is displayed and has no users in it
+        And assignment type of displayed cohort to "manual" because this is the default
         And when I add the user to the new cohort
         Then the cohort has 1 user
         And appropriate events have been emitted
         """
         start_time = datetime.now(UTC)
+        assignment_type = 'manual'
```

Comment:
Nit: Since you don't pass assignment_type on line 271 (which I understand, since you want to test the default behavior), I'd just explicitly compare to "manual" on line 281 instead of creating this variable.